packages/nextjs/components/scaffold-eth/Contract/utilsContract.tsx

```diff
@@ -160,6 +160,24 @@ const getParsedEthersError = (e: any): string => {
 
   return message;
 };
+const extractKeyValueErrorPairs = (errorMessage: any) => {
+  let keyValuePairs = {} as any;
```

Comment:
Don't assert as `any`, be as strict as possible.
Also assigning an empty object as default value is probably not the best way to go, I think.

I'd remove this `variable` and declare it inside the `try { ... } catch` block. Handle the rest of the logic there.

If the logic fails (due to invalid JSON or for whatever other reason), the default fallback of this method should be to return the error message as is.